src/App.css

```diff
@@ -25,7 +25,9 @@
 
 :root {
   /* Seems to fix a Safari glitch: https://github.com/microbit-foundation/python-editor-next/issues/369 */
```

Comment:
Will this change cause this issue to appear again? https://github.com/microbit-foundation/python-editor-next/issues/369

Reply:
No longer changed in theme approach.